src/exceptions/process-error-handler.ts

```diff
@@ -0,0 +1,15 @@
+import { Logger } from 'winston';
```

Comment:
Great, appreciate it!
But could you move it in index file to explicitly handle all top level errors?